cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/UserManagementTest.scala

```diff
@@ -41,48 +41,48 @@ class UserManagementTest extends AdministrationCommandTestBase {
 CREATE USER alice SET PASSWORD $password
 ###
 
-Create a user with the username `alice` that needs to change her password on first login.
+Create a new user and a password. This password must be changed on the first login.
 
 ###assertion=update-one parameters=update
 //
 
 ALTER USER alice SET PASSWORD $password CHANGE NOT REQUIRED
 ###
 
-(★) Change the password for the user `alice` and update so that she is not required to change password on next login.
+(★) Set a new password for a user. This user will not be required to change this password on the next login.
 
 ###assertion=update-one parameters=update
 //
 
 ALTER USER alice SET STATUS SUSPENDED
 ###
 
-(★) Change the status for the user `alice` so that she is suspended. Use `SET STATUS ACTIVE` to reactivate `alice`.
+(★) Change the user status to suspended. Use `SET STATUS ACTIVE` to reactivate the user name.
 
 ###dontrun
 // Can't be run since we can't log in as a user, and have auth disabled
 
 ALTER CURRENT USER SET PASSWORD FROM $old TO $new
 ###
 
-Change the password for the logged in user and update so that a password change is not required on next login.
+Change the password of the logged-in user, and update. The user will not be required to change this password on the next login.
 
 ###assertion=show-one
 //
 
 SHOW USERS
 ###
 
-List all users in the system, their status, roles and if they need to change their password.\n
-(★) Status and roles are enterprise only.
+List all users in the system, their status, roles and if they need to change their password. +
```

Comment:
We wanted to have a line break here but did not know how to, don't know what the `+` does either :P

Reply:
It's the `+` that starts the new line 😄 

I'll see if I can add them where it makes sense to start new lines.